src/Controller/WopiController.php

```diff
@@ -65,7 +65,7 @@ function wopiCheckFileInfo(string $id, Request $request) {
             'UserId' => $jwt_payload->uid,
             'UserFriendlyName' => $user->getDisplayName(),
             'UserExtraInfo' => [
-                'avatar' => $avatarUrl,
+                'avatar' => $avatarUrl ?? NULL,
```

Comment:
I think the value `avatar` shouldn't be added if there is no avatar.

Reply:
It depends what Collabora expects to find in a request like this.
The proposed change removes the PHP warning, but still sends the same data to Collabora as before.
Sure we can remove the array key if it is NULL, if you say this is ok for Collabora.